src/src/main.c

```diff
@@ -65,6 +68,10 @@ void setup(void)
 
   status_leds_init();
 
+#ifndef LED_INDICATION_OF_VTX_MODE
+  resetModeIndication()
```

Comment:
Missing ";"

Reply:
indeed, sorry for that. it also should be #ifdef instead. I've pushed a fix for that